testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestSyncPartitionMetadata.java

```diff
@@ -104,12 +102,11 @@ public void testAddPartitionContainingCharactersThatNeedUrlEncoding()
         cleanup(tableName);
     }
 
-    @Test(groups = {HIVE_PARTITIONING, SMOKE})
     @Flaky(issue = ERROR_COMMITTING_WRITE_TO_HIVE_ISSUE, match = ERROR_COMMITTING_WRITE_TO_HIVE_MATCH)
```

Comment:
is `@Flaky` still applied, if the test method is overridden?

Reply:
My bad. No, it's not applied in case of override.